api/swagger.yml

```diff
@@ -610,6 +610,8 @@ definitions:
       currentStep:
         type: integer
         format: int64
+      html_url:
```

Comment:
I think all (or almost all) JSON objects (both args and responses) in the swagger file follow camelCase naming convention. Shall we name it `htmlUrl` instead? @gordonsyme how do you think?

Reply:
Sure, consistency matters 👍